### PR TITLE
update with end of 2016 events

### PR DESCRIPTION
**Pull request by anthonydahanne**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -6,16 +6,43 @@ permalink: /events/
...
+The Terracotta / Software AG team will be out in full force.  Join us at our booth, see demos, get technical assistance, and win prizes.
+
+Also, don't forget to catch talks given by the Ehcache / Terracotta / Software AG crew!
+
+#### Ehcache and Terracotta Server talks at Devoxx Belgium 2016
+
+- [Ehcache out of his element](http://cfp.devoxx.be/2016/talk/AUX-3854/Ehcache_out_of_his_Element), by [Louis Jacomet](http://cfp.devoxx.be/2016/speaker/louis_jacomet)
+- [Terracotta Ehcache: Simpler, faster, distributed](http://cfp.devoxx.be/2016/talk/EMH-3283/Terracotta_Ehcache:_simpler,_faster,_distributed), by [Anthony Dahanne](http://cfp.devoxx.be/2016/speaker/anthony_dahanne)
+
+### Devoxx Morocco : November 1st 2016 : Casablanca, Morocco
```
Comment (ljacomet) [inline index.md]: Content is incorrect - get rid of the section for now, will do a separate PR later

File: index.md [hunk 2]
```diff
@@ -6,16 +6,43 @@ permalink: /events/
...
+
+Louis Jacomet will be making a presentation on adding scale and performance to applicaitons.  
+- Ehcache out of his element, by Louis Jacomet
 
 ### Distributed Systems Meet-up Group : Oct 22, 2016; 10:00 AM - 1:00 PM : Bangalore, India
 
 Join us at [Bangalore-Distributed-Systems-Meetup](http://www.meetup.com/Bangalore-Distributed-Systems-Meetup/) on Meetup.com
 
 #### This months's AGENDA -
```
Comment (ljacomet) [inline index.md]: Get rid of the month reference

File: index.md [hunk 3]
```diff
@@ -6,16 +6,43 @@ permalink: /events/
 active_sub_menu_id: shared_mnu_events_events
 ---
 
-## Upcoming Conferences & Events
-Meet our team and join our sessions at the following conferences and meetups.
+## Recent Events
+
+### Confoo Vancouver : December 7th 2016
```
Comment (ljacomet) [inline index.md]: Missing all commas to get "month date, year" style used elsewhere.

**Review by ljacomet — CHANGES_REQUESTED** [review]
See individual comments

File: index.md (hunk 3 above)
Comment (anthonydahanne) [reply index.md]: okee

File: index.md (hunk 1 above)
Comment (anthonydahanne) [reply index.md]: okee

File: index.md (hunk 2 above)
Comment (anthonydahanne) [reply index.md]: okee

**Comment by anthonydahanne** [comment]
updated PR after comments

**Review by ljacomet — APPROVED** [review]

**Pull request merged**